dashed/models.py

```diff
@@ -68,6 +69,22 @@ def created_by_(self):
     def changed_by_(self):
         return '{}'.format(self.changed_by or '')
 
+    @property
+    def modified(self):
+        s = humanize.naturaltime(datetime.now() - self.changed_on)
+        return "<nobr>{}</nobr>".format(s)
```

Comment:
I removed these from the dashboard page, they are non-standard DOM elements and are not on the roadmap to be supported. maybe use something else? `<span>` or something?

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nobr